### PR TITLE
[1.1.x] Configs for Prusa i3 Pro B

### PR DESCRIPTION
**Pull request by eine**
Two examples are provided, one without any probe and another one with BLTOUCH/3DTOUCH.

### CODE REVIEW

**Comment by thinkyhead** [comment]
Is the BLTouch a standard add-on for this machine, or as a third-party add-on does it have a standardized mount?

**Comment by eine** [comment]
@thinkyhead, you are so fast! Than you!

I have seen that 2-3 different mounts are used for the BLTouch, depending on other (optional) parts, such as a layer cooling fan. However, these configurations are based on http://www.geeetech.com/wiki/images/6/61/3DTouch_auto_leveling_sensor-1.zip, which is the one suggested by Geeetech. Then, I'd say that, yes, this is the standard add-on.

Anyway, there is a README in the folder with minimum instructions, where this is explicitly said. Should I reword it to make it clear that different mounts would require different offset settings?

**EDIT**

I saw that you renamed the folder to `Prusa i3 Pro B`. I am ok with removing the board name, because it is explained in the README and described in the config file. However, isn't it better to avoid using spaces in the folder name? I.e. `Prusa_i3_Pro_B`. Or `I3_Pro_B`, to be consistent with the existing `I3_Pro_X-GT2560`.

**Comment by thinkyhead** [comment]
> However, isn't it better to avoid using spaces in the folder name?

I think it's perfectly safe in 2018 to use spaces in file and folder names.

**Comment by eine** [comment]
> I think it's perfectly safe in 2018 to use spaces in file and folder names.

Sure. Yet, it is not about safety, but about ease of use and coherence.

On the one hand, in spanish ISO layouts, writing `\` to allow autocompletion (`TAB`) is harder than writing `-` or `_`.

On the other hand, spaces are avoided in almost every other subdir in `example_configurations`: `Hephestos_2`, `kossel_mini`, `kossel_pro`, `kossel_xl`, `i3-2020`, `I3-Pro-X-GT2560`, `i3-M508`... I found a single case with a space: `Duplicator 6`.

**Comment by thinkyhead** [comment]
I must admit, I so seldom ever need to actually type out a file path with my monkey fingers that I would never run into these space issues. And the terminals I use include backslashes when auto-completing. Regrettably I have too little time and have to move on from this debate.
